cmd/minishift/cmd/config/util.go

```diff
@@ -70,3 +71,9 @@ func SetBool(m MinishiftConfig, name string, val string) error {
 	m[name] = b
 	return nil
 }
+
+func SetSlice(m MinishiftConfig, name string, val string) error {
+	b := strings.Split(val, ",")
```

Comment:
Is this also called on rereading the file?
If that is the case, just also split on space... ? just like splitting on strings... or am I missing something (haven't been able to run the code yet)